pynxtools/dataconverter/convert.py

```diff
@@ -34,18 +34,14 @@
 from pynxtools.dataconverter.template import Template
 from pynxtools.nexus import nexus
 
+from pynxtools.dataconverter.logger import logger as pynx_logger
+
 if sys.version_info >= (3, 10):
     from importlib.metadata import entry_points
 else:
     from importlib_metadata import entry_points
 
 
-logger = logging.getLogger(__name__)  # pylint: disable=C0103
-UNDOCUMENTED = 9
-logger.setLevel(logging.INFO)
-logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
Why has the `UNDOCUMENTED` level been removed?
I introduced to be able to separate info and undocumented loggings.